web/satellite/src/views/Dashboard.vue

```diff
@@ -67,7 +67,7 @@
             await this.$store.dispatch(PROJETS_ACTIONS.SELECT, getProjectsResponse.data[0].id);
 
             await this.$store.dispatch(PM_ACTIONS.SET_SEARCH_QUERY, '');
-            const projectMembersResponse = await this.$store.dispatch(PM_ACTIONS.FETCH);
+            const projectMembersResponse = await this.$store.dispatch(PM_ACTIONS.FETCH, 1);
```

Comment:
magic number

Reply:
This will be replaced with const after rewriting to class component. Not in this task scope